### DISCUSSION TITLE
Is it possible to run danger on workflow_run?

### DISCUSSION THREAD

**Discussion by H01001000** [danger/danger-js#1373] [Q&A]
Hi, is it possible to have two danger, one trigger on pr and one trigger on workflow_run, which the one trigger on pr upload all info as artifact and use workflow_run one to post comment?

**Comment by orta** [comment]
Yep, you have to use the GitHub APIs to do all that but there's nothing stopping you

**Reply by H01001000** [reply]
Is it a way that I can set the pr I am commenting on? Since in workflow run, there is no pr info for danger

**Reply by orta** [reply]
No, you'd just get the runtime with everything but `danger.github.pr` set up right now - I think I'd be open to a PR which supports a known set of params to the workflow run to know which PR to run on

**Reply by H01001000** [reply]
Is there any way I can comment message on the result of a pr's action check? Thanks

**Reply by orta** [reply]
Sure, use the `danger.github.api` to do whatever you want with the GitHub API
